sidekick/cli_template/bricks/package/__brick__/tool/install.sh

```diff
@@ -8,22 +8,15 @@ cd "${CLI_PACKAGE_DIR}" || exit
   echo "Installing {{name}} command line application..."
 
   # export pub from .flutter dir
-  REPO_ROOT=$(git rev-parse --show-cdup)
   DART_SDK="${CLI_PACKAGE_DIR}/build/cache/dart-sdk"
-  DART="$DART_SDK/bin/dart" 
+  DART="$DART_SDK/bin/dart"
 
   # If we're on Windows, invoke the batch script instead
   OS="$(uname -s)"
   if [[ $OS =~ MINGW.* || $OS =~ CYGWIN.* ]]; then
     DART="$DART_SDK/bin/dart.exe"
   fi
 
-  # Download Dart runtime
-  if [ ! -d "$DART_SDK" ]; then
-    echo 'missing flutter sdk'
-    sh "${CLI_PACKAGE_DIR}/tool/download_dart.sh"
-  fi
-
   # build
   EXE="build/cli.exe"
   printf -- "- Getting dependencies\n"
```

Comment:
Yes, we should try removing it and always run with `set -e`. I don't remember the exact reason. It may be because of null-safety where dart was throwing warnings and exit with non-zero, even though everything was fine.